### PR TITLE
registry/CON-3906: Added documnentation for new registry/http endpoints: ListRepositoriesV2 and ListManifests

### PR DESCRIPTION
**Pull request by CollinShoop**
- Adds docs for repository `ListManifests` endpoint
- Adds docs for repository `ListRepositoriesV2` endpoint
- Deprecates `ListRepositories` endpoint

### CODE REVIEW

**Comment by andrewsomething** [comment]
!deploy

File: specification/DigitalOcean-public.v2.yaml [hunk 1]
```diff
@@ -1124,10 +1124,14 @@ paths:
     post:
       $ref: 'resources/registry/validate_registry_name.yml'
 
-  /v2/registry/{registry_name}:
+  /v2/registry/{registry_name}/repositories:
     get:
       $ref: 'resources/registry/list_registry_repositories.yml'
```
Comment (andrewsomething) [inline specification/DigitalOcean-public.v2.yaml:1129]: If we're going to be deprecating v1 of this endpoint, we'll want to add `deprecated: true` to  `resources/registry/list_registry_repositories.yml`

Comment (CollinShoop) [reply specification/DigitalOcean-public.v2.yaml:1129]: Added, along with some additional language to prompt users to instead of the [V2] endpoint.

**Review by andrewsomething — APPROVED** [review]
:+1:  Looks great!

**Comment by CollinShoop** [comment]
The http endpoints being documented are now available.

**Pull request merged**